releases/firefox-release-rc-49.0.json

```diff
@@ -6,12 +6,11 @@
             "graphid": "",
             "graphid_2": "",
             "human_tasks": {
-                "post_released": false,
-                "published_balrog": false,
+                "published_release": false,
                 "pushed_mirrors": false,
                 "submitted_shipit": false,
                 "emailed_cdntest": false,
-                "published_beta_balrog": false
+                "published_rc_beta_release": false
```

Comment:
it was following the convention of published_release. but to point out that it was published_RC_BETA_release.

but I'll change it to what you suggested